INSTALLATION.md

```diff
@@ -81,7 +81,8 @@ Copy/paste the endpoint for accessing talawa-api graphql service to the variable
 
 ## Setting up REACT_APP_RECAPTCHA_SITE_KEY in .env file
 
-Make sure to go through [this](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md#setting-up-recaptcha_secret_key-in-env-file) section on `talawa-api's` installation docs.
+Make sure to go through INSTALLATION.md  `setting-up-recaptcha_secret_key-in-env-file` section of [Talawa-API repo](https://github.com/PalisadoesFoundation/talawa-api).
+
```

Comment:
"Refer to to the `RECAPTCHA`  section in the INSTALLATION.md file found in the [Talawa-API repo](https://github.com/PalisadoesFoundation/talawa-api)"

Please update in all the places reported